osquery/dispatcher/scheduler.cpp

```diff
@@ -100,7 +101,11 @@ SQLInternal monitor(const std::string& name, const ScheduledQuery& query) {
 
 Status launchQuery(const std::string& name, const ScheduledQuery& query) {
   // Execute the scheduled query and create a named query object.
-  VLOG(1) << "Executing scheduled query " << name << ": " << query.query;
+  if (!FLAGS_verbose) {
```

Comment:
Ah, I recommend adding a new flag for this `FLAGS_schedule_lognames`.

Reply:
Then the logic would be:

```
if (FLAGS_verbose) {
  VLOG(1) << "Executing scheduled query " << name << ": " << query.query;
} else if (FLAGS_schedule_lognames) {
  LOG(INFO) << "Executing scheduled query " << name;
}
```

Reply:
Done